### DISCUSSION TITLE
比龙猫做的还好，建议与UP商量一下

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#3306] [讨论]
[https://www.bilibili.com/video/BV1nC41147Zd/?spm_id_from=333.1007.tianma.4-2-12.click&vd_source=befe21dae916c45f4b101ad81a3785da](url)

**Comment by allMagicNB** [comment]
![Screenshot_20240211_075510.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/07e2fb37-4256-4766-b5d3-f1a190e6f4c7)

# 这里是启动器仓库，龙猫又没怎么想着做宣传，请不要觉得好玩而乱提建议

**Comment by Enigma-Soul** [comment]
兄弟你不识字吗
![image](https://github.com/Hex-Dragon/PCL2/assets/119654171/00a7498e-76af-4f0b-bfca-1a78a4a9fbc3)

**Comment by allMagicNB** [comment]
> 兄弟你不识字吗
> ![image](https://github.com/Hex-Dragon/PCL2/assets/119654171/00a7498e-76af-4f0b-bfca-1a78a4a9fbc3)
> 

你这不是建议吗？

**Reply by Enigma-Soul** [reply]
讨论，要不是贴吧被龙猫关了，我早就去爱发电的社区了

**Reply by allMagicNB** [reply]
> 讨论，要不是贴吧被龙猫关了，我早就去爱发电的社区了
> 

那不也是类型为讨论的建议（

**Comment by Pigeon0v0** [comment]
丢两张图，自己看看什么成分吧

接下来不要继续在这里讨论相关这位的事情

![image](https://github.com/Hex-Dragon/PCL2/assets/60414767/4273889c-601b-4da4-9ae4-d413d46844e7)

![image](https://github.com/Hex-Dragon/PCL2/assets/60414767/d5252fe4-e329-404d-b962-e64c822f0717)

**Reply by Deep-Dark-Forest** [reply]
是的甚至java版标题屏幕的右下角的文字还由英文变成了中文，这下可让他看明白了（

**Reply by Pigeon0v0** [reply]
> 是的甚至java版标题屏幕的右下角的文字还由英文变成了中文，

吃完饭砸碗骂厨子 我反正不好评 忍不住还是说一句

**Reply by ghost** [reply]
![Screenshot_20240214_151117](https://github.com/Hex-Dragon/PCL2/assets/139457095/f230e120-a654-45ef-8bca-ae945176309c)
我无法确定……是我误解了谁的意思吗

**Reply by Pigeon0v0** [reply]
> 我无法确定……是我误解了谁的意思吗

那他大可以把 BMCLAPI 也举报了，做个彻头彻尾的正义使者，大家都去用加速器吧

Mojang 这么久都没管国内镜像的原因我想你应该也清楚

**Reply by Deep-Dark-Forest** [reply]
@Pigeon0v0 这个屑up的置顶动态的第十三行（手动换行算一行）暴露了他自己是一个骂“盗版”还去用“盗版”的小丑，然后我在他的动态下评论还给我删了哈哈哈哈嗝

**Reply by allMagicNB** [reply]
连真实用户名都不敢泄露，泄露出来的用 NameMC 查都查不到，那些截图能证明什么？Or Photoshop Boss?成就截屏一点都没用，随便找个下载站都能下载到 Android 版，成就也能刷啊。
而且一个披风，只要参与过投票就有，还得宣传出来？
Among US 要钱吗？捣蛋鹅要钱吗？剩下的游戏都有盗版，怎么证明？
购买过正版 Windows 11？我看是密钥吧，密钥才几个钱，一套正版都要一千，笔记本出厂还自带正版，你这正版还会报错呢？

**Comment by allMagicNB** [comment]
![R-C.jpeg](https://github.com/Hex-Dragon/PCL2/assets/116283787/22c5fb92-7f37-4d65-8157-ee9561713ded)
# 您先照照镜子看看自己是啥

**Reply by iPodtouch6557** [reply]
_

**Reply by allMagicNB** [reply]
> 一个圆而已，照不出啥

你等我换图片（

**Comment by iPodtouch6557** [comment]
noisy =>lock as off-topic
:)

**Reply by MoYuan-CN** [reply]
good idea :)

**Reply by allMagicNB** [reply]
Looks so nice
=)

**Comment by zkitefly** [comment]
我认为需要将此讨论锁定。。。

**Reply by allMagicNB** [reply]
要不然删除？

**Reply by iPodtouch6557** [reply]
马上锁，~这个决定实在是太伟大，英明，正确了~

**Reply by allMagicNB** [reply]
> 马上锁，~这个决定实在是太伟大，英明，正确了~

I think this is a right choice.
